vuu-ui/showcase/src/examples/Apps/NewTheme.examples.tsx

```diff
@@ -235,7 +176,6 @@ const ShellWithNewTheme = () => {
         }
         loginUrl={window.location.toString()}
         user={user}
-        saveLocation="local"
```

Comment:
This has just reminded me of the hole environment variables stuff. Did we come to a solution for that? I'm just wondering how will a developer know how to set the implementation to remote/local? Are we tracking the need for documentation anywhere?